lib/accounting/adapters/xero_adapter.ex

```diff
@@ -57,20 +71,22 @@ defmodule Accounting.XeroAdapter do
 
     {:ok, tracking_category_id}
   end
-  defp ensure_tracking_category_exists({:ok, %{status_code: 404}}) do
+  defp ensure_tracking_category_exists({:ok, %{status_code: 404}}, credentials) do
     "start_link.xml"
     |> render()
-    |> put("TrackingCategories", 5_000)
-    |> ensure_tracking_category_exists()
+    |> put("TrackingCategories", 5_000, credentials)
```

Comment:
It’s referenced only in `ensure_tracking_category_exists`, which is used by `start_link`. It’s a simple, small request that should return a response within 5 seconds. If it doesn’t, a crash is appropriate.